src/edu/csus/ecs/pc2/clics/API202306/EventFeedJSON.java

```diff
@@ -1056,13 +1069,80 @@ public void setFilter(Filter filter) {
         this.filter = filter;
     }
 
+    /**
+     * Check if we are using collections for notifications
+     *
+     * @return true if collections are enabled, false otherwise
+     */
     public boolean isUseCollections() {
         return useCollections;
     }
 
+    /**
+     * Set if we want to try to use collections when possible for notifications
+     *
+     * @param useCollections - true to try to use collections
+     */
     public void setUseCollections(boolean useCollections) {
         this.useCollections = useCollections;
     }
+
+    /**
+     * Tests if notification collections should be used for a specific notification type.
+     * This first tests is notification collections are being used in general, then if so,
+     * checks if the specific type is not disabled.
+     *
+     * @param notificationType
+     * @return
+     */
+    public boolean isUseNotificationCollection(EventFeedType notificationType) {
+        return(isUseCollections() && !disableCollections.contains(notificationType.toString()));
+    }
+
+    /**
+     * Adds the comma-separated list of notification types (EventFeedType's) to the HashSet of
+     * disabled notification types.  Any notification type in this list will not use collections
+     * on the event feed.  The special type ALL_NOTIFICATION_TYPES will disable collections for all
+     * notification and will clear the HashSet of disabled types and simply set the useCollections flag to false.
+     *
+     * @param list Comma-separated list of notification types (EventFeedType's) to not use
```

Comment:
As mentioned above, the actual parameter here is not a comma-separated list; I think this is confusing.

Reply:
Fixed in next push.